styles.css

```diff
@@ -82,19 +71,38 @@ nav .bar1,
 
 .icons-container > ul {
   display: flex;
-  justify-content: space-between;
   max-width: 50%;
   margin-left: 10px;
   flex-direction: row;
   list-style-type: none;
 }
```

Comment:
- [ ] kindly consider adding some padding on this section as it was on mobile version
![ma1](https://user-images.githubusercontent.com/60146030/220933192-ccd9bb27-1608-4507-ac18-c64555fa9653.PNG)